app/assets/javascripts/search.js

```diff
@@ -16,12 +18,22 @@
       $this = $(this);
       formData = $(this).serialize();
       history.replaceState(null, '', "?" + formData);
-      return delay((function() {
-        $this.submit();
-        return $this.on('ajax:success', function(e, data, status, xhr) {
+
+      debounce(function() { 
+        $this.one('ajax:beforeSend', function(e, data, status, xhr) {
+          $this.find(":input").prop("disabled", true);
+        });
+
+        $this.one('ajax:complete', function(e, data, status, xhr) {
+          $this.find(":input").prop("disabled", false);
```

Comment:
How come you are using `"` in some places but `'` in others? I don't know JS very well but are they interchangeable like this? And if they are interchangeable, should we strive for consistency?

Reply:
Yes, they're interchangeable. And you're so right, I usually have my editor settings turned on to ensure consistency but it seems I forgot to enable that for my work computer. Changes incoming